servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1326,8 +1335,21 @@ public void reportLostFile(long fileId) throws FileDoesNotExistException {
     return mDependencyMap.getPriorityDependencyList();
   }
 
+  /**
+   * Loads metadata for the object identified by the given path from UFS into Tachyon.
+   *
+   * @param path the path for which metadata should be loaded
+   * @param recursive whether parent directories should be created if they do not already exist
+   * @return
+   * @throws BlockInfoException if an invalid block size is encountered
+   * @throws FileAlreadyExistsException if the object to be loaded already exists
+   * @throws FileDoesNotExistException if a parent directory does not exist and recursive is false
+   * @throws InvalidPathException if invalid path is encountered
+   * @throws SuspectedFileSizeException if invalid file size is encountered
+   * @throws IOException if an I/O error occurs
+   */
   // TODO(jiri): Make it possible to load directories and not just individual files.
```

Comment:
The TODO's goal is to load UFS objects recursively (not just individual objects which is the current functionality). I can see how the TODO was confusing though, so I updated it.